src/autoscaler-common/counters_base.js

```diff
@@ -73,27 +73,45 @@ const COUNTERS_PREFIX =
 
 /** @enum{String} */
 const ExporterMode = {
-  GCM: 'GCM',
-  OTEL: 'OTEL',
+  GCM_ONLY_FLUSHING: 'GCM_ONLY_FLUSHING',
+  OTEL_PERIODIC: 'OTEL_PERIODIC',
+  OTEL_ONLY_FLUSHING: 'OTEL_ONLY_FLUSHING',
 };
 
 const EXPORTER_PARAMETERS = {
-  // In GCM direct push mode, we never want to export automatically
-  // and we want to rely only on flushing.
-  // This is to avoid that the auto-push happens just after a flush
-  // and then raises 'points were written more frequently' errors.
+  // GCM direct pushing is only done in Cloud functions deployments, where
+  // we only flush directly.
   //
-  // However the only Metrics pusher is a periodic one :/
-  [ExporterMode.GCM]: {
-    PERIODIC_FLUSH_INTERVAL: 12 * 60 * 60 * 1_000, // 12hr
+  [ExporterMode.GCM_ONLY_FLUSHING]: {
+    PERIODIC_FLUSH_INTERVAL: 0x7fffffff, // approx 24 days in milliseconds
```

Comment:
To match the comment below, would it help to distinguish between the two modes of operation by using `PERIODIC_EXPORT_INTERVAL` or `BATCH_EXPORT_INTERVAL` or similar to contrast this with the use of "flush" elsewhere?

Reply:
Done